.azure/pipelines/ci.yml

```diff
@@ -18,12 +18,18 @@ pr:
     include:
     - '*'
 
-# Choose whether to skip tests when running pipeline manually.
 parameters:
+# Choose whether to skip tests when running pipeline manually.
 - name: skipTests
   default: false
   displayName: Skip tests?
   type: boolean
+# Choose whether to test source indexing. Ignored in public builds.
+# Will cause inaccessible links on https://source.dot.net/ unless commits are also available in GitHub.
+- name: testSourceIndexing
+  default: false
+  displayName: Test source indexing? !Danger! see comments in YAML.
+  type: boolean
```

Comment:
This parameter is visible in the AzDO UI but, as mentioned in the comments, ignored in public builds.